CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 5.0.0 (2021-10-01)
+
+### New features
+
+- Migrated to Angular CLI. [#180](https://github.com/blackbaud/skyux-tabs/pull/180)
+- Added support for "partial" Ivy compilation mode. [#184](https://github.com/blackbaud/skyux-tabs/pull/184)
+- Added support for `@angular/core@^12`. [#169](https://github.com/blackbaud/skyux-tabs/pull/169)
```

Comment:
We have been wording this one as "Added support for Angular 12." in the other repos on the major release.

The first 2 we haven't been adding as they are just kind of thrown together with this entry and we don't need the last one here as there isn't really anything there that helps consumers. Most of the `New features` sections have only had the one entry.

Reply:
See:

https://github.com/blackbaud/skyux-data-manager/pull/53/files
https://github.com/blackbaud/skyux-phone-field/pull/102/files

Reply:
fixed in commit: [3687161 (fix changelog wording)](https://github.com/blackbaud/skyux-tabs/pull/194/commits/3687161cd0f07cd675edf3692bc1aac883c50501)